crates/strategy/zksync/src/executor/runner.rs

```diff
@@ -73,6 +119,271 @@ impl ExecutorStrategyRunner for ZksyncExecutorStrategyRunner {
         Ok(())
     }
 
+    fn get_nonce(&self, executor: &mut Executor, address: Address) -> BackendResult<u64> {
```

Comment:
As per https://github.com/matter-labs/foundry-zksync/pull/839 we should only use either `full_nonce`, `transaction_nonce`, or `deployment_nonce` within zksync context to disambiguate them.

Reply:
This one is shared with EVM strategy, the zksync-specific one is aptly named